sar_pipeline/aws/preparation/burst_utils.py

```diff
@@ -92,9 +116,9 @@ def get_burst_ids_and_start_times_for_scene_from_asf(
 
     Returns
     -------
-    tuple[list[str],list[datetime]]
-        List of burst ids. e.g. ['t070_149822_IW3','t070_149822_IW2' ....]
-        List of sensing start times corresponding to each above burst ids.
+    dict
+        unique dict of scene burst ids mapped to start_times and geometries
+        {'t070_149822_IW3' : {start_time : datetime.datetime, geometry: shapely.geom}}
```

Comment:
Code below that creates the dictionary appears to include the polarisation as well. Suggest updating this after updating the type hinting.